repo-for-binabox-main/contact_us.html

```diff
@@ -0,0 +1,113 @@
+                <div class="nav-item2"><a herf="#">ABOUT</a></div>
+                <div class="nav-item3"><a herf="#">ROADMAP</a></div>
+                <div class="nav-item4"><a herf="#">COLLECTION</a></div>
+                <div class="nav-item5"><a herf="#">FAQS</a></div>
+                <div class="nav-item6"><a herf="#">PAGES</a></div>
+            </ul>
+        </nav>
+    
+        <div class="icons">
+            <div class="sun4">
+                <img src="images/sun.svg" alt="sun image">
+            </div>
+            <button><a href="https://discord.com/"><p>Discord</p></a></button>
+            <button type="button"><img src="images/holder.png">Connect</button> 
+        </div>
+    </header>
+    <main>
+        <div class="contact_us_main_head">
+            <h1>CONTACT US</h1>
+            <p>Home</p><p>Contact Us</p>
+        </div>
+        <div class="contact_us_main_cards">
+            <div class="contact_us_main_cards_location">
+                <img src="" alt="location">
+                <h2>location</h2>
+                <p>2163 Phillips Gap Rd West Jefferson</p>
+            </div>
+            <div class="contact_us_main_email">
+                <img src="" alt="email">
+                <h2>Email</h2>
+                <p>Info.avitex@gmail.com</p>
+            </div>
+            <div class="contact_us_main_phone">
+                <img src="" alt="phone">
+                <h2>Phone</h2>
+                <p>+1 666 8888</p>
+            </div>
+        </div>
+        <div class="contact_us_main_getintouch">
+            <h1>Get in touch</h1>
+            <p>Browse through the most freuently asked questions</p>
+            <div class="contact_us_main_getintouch_inputbar">
+
+                <input class="form-text-input" type="text" placeholder="Name">
+
+                <input class="form-text-input" type="email" placeholder="Enter your email">
+
+                <input class="form-text-input" type="tel" placeholder="Phone Number">
+
+                <input class="form-text-input" type="text" placeholder="Message">
+
+                <button>SEND MESSANGER</button>
+            </div>
+        </div>
+        <div class="contact_us_main_join_us">
+            <h2>Join our community</h2>
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. </p>
+            <button><img src="images/discord logo.png" alt="discord logo"><a href="discord.com">Discord</a></button>
+            <button type="button">Whitelist Now</button>
+        </div>
+    </main>
+    <footer>
+        <div class="footer2.1">
+            <img src="images/binabox.svg" alt="Binabox logo"> <h2>BINABOX</h2>
+            <p>We are the best way to check the rarity of NFT collection.</p>
+            <h3><a href="SUBSCRIBE">SUBSCRIBE</a></h3>
+            <a href="Home">Home</a>
+            <a href="About">About</a>
+            <a href="Roadmap">Roadmap</a>
+            <a href="Our Team">Our Team</a>
```

Comment:
Could you please use links in `href` attribute? Basically it is file names of sibling files. If there's no files for some pages, you can leave attribute with empty anchor value (`#`). Click on it will not reload the page and will not lead to any another page.
I want to highlight that it's invalid to use space in paths, so w3c-validator will show an error here.